docs/topics/proc_deploying-a-node-js-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,45 @@
+[id='deploying-a-node-js-application-to-standalone-rhel_{context}']
+= Deploying a {Node} application to standalone Red Hat Enterprise Linux
+
+Deploy a {Node} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* A {Node} application.
+* npm 6.4.1 installed
```

Comment:
This document does not specify a version for Node.js, so why are we specifying one for npm? We produce RPMs for both of them, and the versioning typically reflects a pair of binaries.

Reply:
Yo're right It does look inconsistent this way. The required npm version I also documented in the release notes, so IMHO it doesn't have to be listed here, too. we could just say 

> npm installed

as the prerequisite.